src/pages/Result/Result.jsx

```diff
@@ -0,0 +1,119 @@
+          {Number(score) >= 4 && (
+            <>
+              <img
+                className="prize-img"
+                src={ResultTrophy}
+                alt="result-trophy"
+              />
+              <div>Congratulations!!!!</div>
+            </>
+          )}
+          {Number(score) === 3 && (
+            <>
+              <img className="prize-img" src={goodtry} alt="goodtry" />
+              <div>Good Try!!!!</div>
+            </>
+          )}
+          {Number(score) <= 2 && (
+            <>
+              <img className="prize-img" src={oops} alt="oops" />
+              <div>Better Luck Next Time!!!</div>
+            </>
+          )}
+          <p className="final-score-info text-lg font-semibold">
+            Final Score: <span>{score * 20}/100</span>
+          </p>
+          {selectedQuizLoading && <Loader />}
+          {!selectedQuizLoading &&
+            selectedQuizData.questions.map((question, questionNumber) => {
+              return (
+                <div
+                  className="result-container flex flex-column flex-align-center flex-gap-2 pd-2"
+                  key={question._id}
+                >
+                  <p className="font-semibold">{question.question}</p>
+                  <ul className="option-container list-style-none flex flex-column flex-align-center flex-gap-1">
+                    {question.options.map((option, index) => {
+                      return option === answers[questionNumber] &&
+                        option === correctAnswers[questionNumber] ? (
+                        <li
+                          key={index}
+                          className="option-label-result correct-ans"
+                        >
+                          {option}
+                        </li>
+                      ) : option === answers[questionNumber] ? (
+                        <li
+                          key={index}
+                          className="option-label-result wrong-ans"
+                        >
+                          {option}
+                        </li>
+                      ) : option === correctAnswers[questionNumber] ? (
+                        <li
+                          key={index}
+                          className="option-label-result correct-ans"
+                        >
+                          {option}
+                        </li>
+                      ) : (
+                        <li key={index} className="option-label-result">
+                          {option}
+                        </li>
+                      );
+                    })}
+                  </ul>
+                </div>
+              );
+            })}
+          <Link
+            to="/"
+            onClick={finishHandler}
+            className="finish-btn btn-solid btn-medium flex flex-justify-center flex-align-center"
+          >
+            Finish
+          </Link>
+        </div>
+      )}
+    </main>
+  );
+};
```

Comment:
Look into those suggestions, rest everything looks good and you can proceed to merge